docs/families/blackcat.md

```diff
@@ -15,13 +15,13 @@ parent: Ransomware Families
 |**Artifacts** | | |
 |Extensions | 7 characters in length, specified in config file | [^5] |
 |Ransomware Notes | RECOVER-[EXTENSION]-FILES.txt | [^5] |
-|Services It Disables | TBD | |
+|Services It Disables | IIS, Antivirus, backup service, recovery tool in Windows boot menu and clear Windows event logs |[^8] |
 |Other Observables | Also named ALPHV and Noberus | [^6]|
 |**Automation** | | |
-|Initial Access | No |  |
+|Initial Access | Yes (exploiting Exchange servers, using remote access tools or getting compromised credentials within dark markets) |[^7] [^8]  |
 |Privilege Escalation | Yes. <br>Several techniques including Masquerade_PEB, UAC bypass via elevated COM interface, and CreateProcessWithLogonW exploit|[^5] |
 |Human Operated | Yes | |
-|Exfiltration | No | |
+|Exfiltration | Yes (through MegaSync, Filezilla and WinSCP) |[^7] |
```

Comment:
That site indicates that the threat actor decided to use megasync and rclone to exfil the data.  Therefore, it is not part of the EXE itself. Our research before indicates that BlackCat does not do exfil and that the attacker has to do it. Did you find other sites that showed BlackCat automatically exfiltrates?